phpunit/blocks/class-wp-navigation-block-renderer-test.php

```diff
@@ -63,6 +63,76 @@ public function test_gutenberg_get_markup_for_inner_block_site_title() {
 		$this->assertEquals( $expected, $result );
 	}
 
+	/**
+	 * Test that the heading block isn't wrapped in a list item to preserve accessible markup
+	 *
+	 * @group navigation-renderer
+	 *
+	 * @covers WP_Navigation_Block_Renderer::get_markup_for_inner_block
+	 */
+	public function test_gutenberg_get_markup_for_inner_block_heading() {
```

Comment:
```suggestion
	/**
	 * Test that a given block will not be automatically wrapped in a list item by default.
	 *
	 * @group navigation-renderer
	 *
	 * @covers WP_Navigation_Block_Renderer::get_markup_for_inner_block
	 */
	public function test_gutenberg_block_not_autonatically_wrapped_with_li_tag() {
```

I think we should test that the default is "don't wrap". And also make the test less about the heading block. It might be confusing and misdirect users to wonder about special features of the heading block.

Reply:
I applied your updates :)